test/TestProjects/FirstTest-Typespec/Generated/Docs/CadlFirstTestClient.xml

```diff
@@ -1047,6 +1065,10 @@ var data = new {
         requiredLiteralBool = new {},
         requiredBadDescription = "<requiredBadDescription>",
     },
+    requiredUnknown = new {},
```

Comment:
when unknow in a model, it will be resolved as object?

Reply:
did not get your point. What are you expecting here?

Reply:
I am asking if it is proper to set unknown property to a empty object.

Reply:
We will resolve this in sample enhancement.